boot/freeldr/freeldr/arch/i386/machxbox.c

```diff
@@ -23,6 +23,18 @@
 DBG_DEFAULT_CHANNEL(HWDETECT);
 
 
+BOOLEAN
+XboxFindPciBios(PPCI_REGISTRY_INFO BusData)
+{
+    TRACE("We emulate PCI BIOS here, there are 2 known working PCI buses on an original Xbox\n");
```

Comment:
"TRACE"? I think this could be just turned into a code comment!

Reply:
@HBelusca I fixed that :ok_hand: